### PR TITLE
Logo Update

### PR DESCRIPTION
**Pull request by Fish2**
#### Database Migration
NO

#### Description
Fixed the main SVG fully symmetrical now and also to match Sonarr, Radarr and Lidarr
Added my resource SVG for later use for any one wanting to add to this
Updated all the favicon's and modernised them
Added Read me Header
Removed some un-used image files
All images have been lossless compressed
Fixed all image links

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR

* #

### CODE REVIEW

File: src/NzbDrone.Console/NzbDrone.Console.csproj [hunk 1]
```diff
@@ -54,7 +54,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <PropertyGroup>
-    <ApplicationIcon>Radarr.ico</ApplicationIcon>
```
Comment (galli-leo) [inline src/NzbDrone.Console/NzbDrone.Console.csproj]: Why was this renamed?

File: src/NzbDrone.Host/NzbDrone.Host.csproj [hunk 2]
```diff
@@ -56,9 +56,6 @@
   <PropertyGroup>
     <RunPostBuildEvent>OnOutputUpdated</RunPostBuildEvent>
   </PropertyGroup>
-  <PropertyGroup>
-    <ApplicationIcon>Radarr.ico</ApplicationIcon>
```
Comment (galli-leo) [inline src/NzbDrone.Host/NzbDrone.Host.csproj]: Why was this removed?

File: src/UI/Content/Images/favicon/manifest.json [hunk 3]
```diff
@@ -1,53 +1,18 @@
 {
     "name": "Radarr",
     "icons": [
-        {
-            "src": "/Content/Images/favicon/android-chrome-36x36.png",
```
Comment (galli-leo) [inline src/UI/Content/Images/favicon/manifest.json]: Are these not needed anymore?

Comment (Fish2) [reply src/UI/Content/Images/favicon/manifest.json]: No this is for the older android very early low res phones also android will automatically down res from the high res images

File: src/NzbDrone.Host/NzbDrone.Host.csproj (hunk 2 above)
Comment (Fish2) [reply src/NzbDrone.Host/NzbDrone.Host.csproj]: Thats was removed and re added more inline how the other 2 forks use .\NzbDrone.Host

**Comment by Fish2** [comment]
@galli-leo  Info.plist is right the file is the same name as the imbedded in the Radarr.app

File: src/NzbDrone.Console/NzbDrone.Console.csproj (hunk 1 above)
Comment (Fish2) [reply src/NzbDrone.Console/NzbDrone.Console.csproj]: just to this in line with the other forks
